content/kapacitor/v1.5/guides/continuous_queries.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Kapacitor as a Continuous Query engine
+description: Use Kapacitor for Continuous Queries (CQs).
```

Comment:
May want to enrich this for SEO. "Continous queries" is in the title so it doesn't necessarily need to be called out in the description. What about:

> Use Kapacitor to downsample and process data at scheduled intervals.